modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
```

Comment:
See openshift-docs guidance for language that introduces commands: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command